x-pack/plugins/observability_solution/infra/public/pages/metrics/settings/indices_configuration_panel.tsx

```diff
@@ -105,6 +116,28 @@ export const IndicesConfigurationPanel = ({
           {...metricAliasFieldProps}
         />
       </EuiFormRow>
+      {isMetricAliasChanged && numberOfInfraRules > 0 && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiCallOut
+            data-test-subj="infraIndicesPanelSettingsDangerCalloutUsedByRules"
+            size="s"
+            title={METRIC_INDICES_USED_BY_RULES}
+            color="danger"
+            iconType="warning"
+          >
+            <FormattedMessage
+              id="xpack.infra.sourceConfiguration.metricIndicesUsedByRulesMessage"
+              defaultMessage="There {isOrAre} {numberOfInfraRules} rule{plural} that rely on the current Metric indices setting. Changing this setting may impact the execution of these rules."
```

Comment:
"that rely on this data source setting" - at the very least we can lowercase "metric" but I think we can probably just drop the "Metric indices" reference altogether and simplify it to match whatever is in the other UI (Logs)